package.json

```diff
@@ -67,5 +67,6 @@
     "pretest": "standard",
     "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'",
     "postinstall": "node ./bin/setup.js"
-  }
+  },
+  "hoodie": {"plugins": []}
```

Comment:
you add that to your app which has `hoodie` as its dependency, we don’t need to add it to `hoodie` iteslf